packages/core/components/heading/index.js

```diff
@@ -1,54 +1,71 @@
 /* @flow */
 
-import PropTypes from 'prop-types';
-import styled, { css } from 'styled-components';
+import { Alignment } from "../../common/alignment";
+
+import PropTypes from "prop-types";
+import styled, { css } from "styled-components";
 
 const Heading = styled.h1`
   color: black;
+  text-align: left;
+
+  ${props =>
```

Comment:
I changed this back doing some debugging but I'm definitely not against the destructuring approach if we really like it, the `{({` was difficult to read at first

Reply:
Another option might be accepting a single letter variable since we'll know exactly what we mean if we did
```jsx
 ${p =>
    p.blue && css`
        background: blue
    `};
```